UnitsNet/UnitDefinitions/Density.json

```diff
@@ -161,6 +161,18 @@
           "Abbreviations": [ "g/ml" ]
         }
       ]
+    },
+    {
+      "SingularName": "PoundPerGallon",
```

Comment:
We might need to separate this into PoundPerUsGallon and PoundPerImperialGallon (or just rename it to make it more explicit).

See Volume: [ImperialGallon](https://github.com/angularsen/UnitsNet/blob/master/UnitsNet/UnitDefinitions/Volume.json#L187) and [UsGallon](https://github.com/angularsen/UnitsNet/blob/master/UnitsNet/UnitDefinitions/Volume.json#L220)

Reply:
Darn, I can't believe I forgot about that.  My dad used to track & calculate his gas mileage in miles per imperial gallon.